pwndbg/commands/argv.py

```diff
@@ -94,11 +94,11 @@ class envp_function(gdb.Function):
     def __init__(self) -> None:
         super().__init__("envp")
 
-    def invoke(self, number=0):
-        number = int(number)
+    def invoke(self, number_value: gdb.Value = gdb.Value(0), *args: gdb.Value) -> gdb.Value:
+        number = int(number_value)
 
         if number > pwndbg.gdblib.argv.envc:
-            return pwndbg.gdblib.typeinfo.void
+            return pwndbg.gdblib.typeinfo.void.optimized_out()
```

Comment:
What does this optimized_out do here?

Reply:
It turns the Type into a Value and tells the caller that there is no information about the actual value of the Value :D

The functions expects a `gdb.Value` return instead of a `gdb.Type` one.